microsphere-multiactive-spring-cloud-loadbalancer/src/main/java/io/microsphere/multiple/active/zone/spring/cloud/loadbalancer/CustomizedLoadBalancerAutoConfiguration.java

```diff
@@ -0,0 +1,17 @@
+package io.microsphere.multiple.active.zone.spring.cloud.loadbalancer;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.cloud.loadbalancer.annotation.LoadBalancerClients;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * {@link ZonePreferenceServiceInstanceListSupplier} Auto-Configuration Class
+ *
+ * @author <a href="mailto:warlklown@gmail.com">Walklown<a/>
+ * @since 1.0.0
+ */
+@Configuration(proxyBeanMethods = false)
+@ConditionalOnProperty(value = "microsphere.customized-loadbalancer.enabled", havingValue = "true")
```

Comment:
I suggest the property name `"microsphere.customized-loadbalancer.enabled"` could be renamed to `"microsphere.spring.cloud.loadbalancer.customized"`, since this feature is only used on Spring Cloud LoadBalancner.

Reply:
Get !